uwblueprint/supportive-housing
# Employee Directory Table And Pagination

## GitHub Issue link

[Employee Directory Table And Pagination](https://github.com/uwblueprint/supportive-housing/issues/93)



## Implementation description
- Create routes to get users through pagination and get count of users
- Create frontend component for table using common styling
- Connect to backend routes and pagination components

Sample:
<img width="1514" alt="Screenshot 2023-07-11 at 12 48 16 AM" src="https://github.com/uwblueprint/supportive-housing/assets/64047863/bb59c4ef-ede2-43e1-a8a9-996657500318">




## Steps to test
1. Make a bunch of users and test that you can see them in the table and that the normal pagination functionality works 
2. Ensure that Active users show green, Invited teal, and Deactivated gray
3. Admins have a role of Admin, No 2FA
4. Regular Staff have a role of Non-Admin, No 2FA
5. Relief Staff have a role of Not Admin, 2FA



## What should reviewers focus on?
* 


## Checklist
